app/views/api/guidance/process-diagrams/participant-training-journey-diagrams.md

```diff
@@ -1,91 +1,295 @@
-# Participant training journey diagrams
+# Participant NPQ journey 
 
-These diagrams give an overview of the processes that take place during the different phases of an NPQ participant's training journey. We'll iterate these based on feedback from providers and DfE support staff.   
+These diagrams provide an overview of the key processes that take place during each phase of an NPQ participant’s journey.
+
+<!-- Added: CSS to align content consistently inside each .card -->
+<style>
+.card {
+  border: 1px solid #ddd;
+  padding: 16px;
+  border-radius: 6px;
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+  margin-bottom: 16px;
+  background: #fff;
+}
```

Comment:
Inline css is out unfortunately - causes to many issues with Content Security Policy

Looks like this shouldn't be needed anyway though?